types/block.go

```diff
@@ -538,6 +543,9 @@ func (sh SignedHeader) ValidateBasic(chainID string) error {
 	if sh.Commit == nil {
 		return errors.New("SignedHeader missing commit (precommit votes).")
 	}
+
+	// TODO: Check Version?
```

Comment:
Will this be addressed in a followup issue?

Reply:
Whoops this is actually checked in the state.ValidateBlock so nothing to check here, can remove this comment